go.mod

```diff
@@ -6,26 +6,24 @@ require (
 	github.com/Shopify/sarama v1.24.1
 	github.com/dustin/go-humanize v1.0.0
 	github.com/go-sql-driver/mysql v1.4.1
-	github.com/gogo/protobuf v1.2.1
+	github.com/gogo/protobuf v1.3.1
 	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
 	github.com/golang/mock v1.2.0
-	github.com/golang/protobuf v1.3.2
-	github.com/google/btree v1.0.0 // indirect
+	github.com/golang/protobuf v1.3.3
 	github.com/google/gofuzz v1.0.0
-	github.com/gorilla/mux v1.6.2
-	github.com/grpc-ecosystem/grpc-gateway v1.7.0 // indirect
+	github.com/gorilla/mux v1.7.3
 	github.com/kami-zh/go-capturer v0.0.0-20171211120116-e492ea43421d
-	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
-	github.com/pingcap/errors v0.11.4
-	github.com/pingcap/kvproto v0.0.0-20191118050206-47672e7eabc0
-	github.com/pingcap/log v0.0.0-20190715063458-479153f07ebd
-	github.com/pingcap/parser v0.0.0-20200103153514-95649ba8a872
-	github.com/pingcap/pd v1.1.0-beta.0.20190923032047-5c648dc365e0
-	github.com/pingcap/tidb v1.1.0-beta.0.20191120070053-5a7ecfeb94fd
-	github.com/pingcap/tidb-tools v3.1.0-beta.0.20191227034743-57985f125c52+incompatible
-	github.com/pingcap/tipb v0.0.0-20191120045257-1b9900292ab6
+	github.com/pingcap/check v0.0.0-20191216031241-8a5a85928f12
+	github.com/pingcap/errors v0.11.5-0.20190809092503-95897b64e011
+	github.com/pingcap/kvproto v0.0.0-20200317043902-2838e21ca222
+	github.com/pingcap/log v0.0.0-20200117041106-d28c14d3b1cd
+	github.com/pingcap/parser v3.1.0-beta.1.0.20200318061433-f0b8f6cdca0d+incompatible
+	github.com/pingcap/pd/v3 v3.1.0-beta.2.0.20200312100832-1206736bd050
+	github.com/pingcap/tidb v1.1.0-beta.0.20200319063417-875036363e1e
+	github.com/pingcap/tidb-tools v4.0.0-beta.1.0.20200317092225-ed6b2a87af54+incompatible
```

Comment:
?

Reply:
can ignore the version of tidb-tools, both v3.1 and v4.0's tag is base on the master branch

Reply:
ok